<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>750</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Guitar</string>
  </property>
  <property name="whatsThis">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;芜湖~&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>400</width>
      <height>650</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>4</width>
      <height>15</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QHeaderView::section {
	background: rgb(210, 250, 250);
}</string>
    </property>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>35</number>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>32</number>
    </attribute>
   </widget>
   <widget class="QLabel" name="label_scaleinfo">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>20</y>
      <width>271</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#aa55ff;&quot;&gt;Guitar Scale&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_key">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>190</y>
      <width>140</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 170, 255);</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_tonality">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>190</y>
      <width>140</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 170, 255);</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_turning">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>290</y>
      <width>140</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 170, 255);</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_chord">
    <property name="geometry">
     <rect>
      <x>738</x>
      <y>290</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 170, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>691</x>
      <y>270</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文彩云</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 85, 255);</string>
    </property>
    <property name="text">
     <string>Chord</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>270</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文彩云</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 85, 255);</string>
    </property>
    <property name="text">
     <string>Turning</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>170</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文彩云</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 85, 255);</string>
    </property>
    <property name="text">
     <string>Tonality</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>170</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文彩云</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 85, 255);</string>
    </property>
    <property name="text">
     <string>Key</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_chordinfo">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>330</y>
      <width>190</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 85, 255);
font: 75 10pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>Chord info</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_mark_choice">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>390</y>
      <width>140</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文彩云</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 85, 255);</string>
    </property>
    <property name="title">
     <string>Accidentals</string>
    </property>
    <widget class="QRadioButton" name="radioButton_sharp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>115</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Terminal</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Sharp</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_flat">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>115</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Terminal</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Flat</string>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="comboBox_chordlevel">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>290</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 170, 255);</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_staff">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>540</y>
      <width>361</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 85, 255);
background-color: rgb(212, 214, 255);</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_output">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>600</y>
      <width>361</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(225, 255, 202);</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_convertkey">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>410</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(0, 0, 0, 0), stop:0.52 rgba(0, 0, 0, 0), stop:0.565 rgba(82, 121, 76, 33), stop:0.65 rgba(159, 235, 148, 64), stop:0.721925 rgba(255, 238, 150, 129), stop:0.77 rgba(255, 128, 128, 204), stop:0.89 rgba(191, 128, 255, 64), stop:1 rgba(0, 0, 0, 0));
color: rgb(85, 170, 255);
font: 75 8pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>Key</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_curkey">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>410</y>
      <width>120</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(193, 252, 255);</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_destkey">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>430</y>
      <width>120</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 247, 211);</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>490</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Number</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_curtonal">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>450</y>
      <width>120</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 226, 184);</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_desttonal">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>470</y>
      <width>120</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(239, 210, 255);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_converttonal">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>450</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;微软雅黑&quot;;
color: rgb(85, 85, 255);
background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(255, 255, 255, 255), stop:0.1 rgba(255, 255, 255, 255), stop:0.2 rgba(255, 176, 176, 167), stop:0.3 rgba(255, 151, 151, 92), stop:0.4 rgba(255, 125, 125, 51), stop:0.5 rgba(255, 76, 76, 205), stop:0.52 rgba(255, 76, 76, 205), stop:0.6 rgba(255, 180, 180, 84), stop:1 rgba(255, 255, 255, 0));</string>
    </property>
    <property name="text">
     <string>Tonal</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
